unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/NFTUtils/NFTUtils.cs

```diff
@@ -143,17 +142,30 @@ public static bool TryParseUrn(string urn, out string chain, out string contract
                 var contractStandardSpan = urnSpan.Slice(0, urnSpan.IndexOf(SEPARATOR));
                 urnSpan = urnSpan.Slice(contractStandardSpan.Length + 1);
 
-                // 4: contract address
-                var contractAddressSpan = urnSpan.Slice(0, urnSpan.IndexOf(SEPARATOR));
-                urnSpan = urnSpan.Slice(contractAddressSpan.Length + 1);
-
-                // 5: token id
-                var tokenIdSpan = urnSpan;
-                contractAddress = contractAddressSpan.ToString();
-                tokenId = tokenIdSpan.ToString();
-                chain = chainSpan.ToString();
-
-                return true;
+                // Check if the URN contains "collections-thirdparty"
+                if (contractStandardSpan.ToString().Equals(COLLECTIONS_THIRDPARTY, StringComparison.Ordinal))
+                {
+                    // 4: contract address
+                    var contractAddressSpan = urnSpan;
+                    contractAddress = contractAddressSpan.ToString();
+                    chain = chainSpan.ToString();
+
+                    return true;
```

Comment:
I would move both `return true;` out of the if-else block to have only one exit point and make it easier to read.